modules/serverless-functions-specifying-function-revision.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-on-cluster-builds.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-functions-specifying-function-revision_{context}"]
+= Specifying function revision
+
+When building and deploying a function on the cluster as opposed to locally, you need to point to the correct function revision by specifying the Git repository, branch, location within the repository, and other parameters. In addition to specifying these parameters using the `func.yaml` configuration file, you can use the flags for the `kn func deploy` command.
+
+.Prerequisites
+
+* {pipelines-title} must be installed on your cluster.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed the Knative (`kn`) CLI.
+
+.Procedure
+
+* Deploy your function:
++
+[source,terminal]
+----
+$ kn func deploy --remote \
+                 --build git \
+		 --git-url <repo-url> \
+                 --git-branch <branch> \
+		 --git-dir <function-dir>
+----
++
+--
+** With the `--remote` flag, the build runs remotely.
+** Substitute `<repo-url>` with the URL of the Git repository.
+** Substitute `<branch>` with the Git branch, tag, or commit.
```

Comment:
This should be noted as optional if the function is on the main branch of the repo.